src/puya/ir/models.py

```diff
@@ -100,6 +100,8 @@ def _frozen_data(self) -> object:
 class Constant(Value, abc.ABC):
     """Base class for value constants - any value that is known at compile time"""
 
+    value: object
```

Comment:
shouldn't need this, these classes are value-equatable, don't need to peer inside them.

Reply:
Yep